app/models/user.rb

```diff
@@ -90,4 +91,12 @@ def generate_auth_token
   def profile_picture_size
     errors.add :profile_picture, :profile_picture_size_limit if profile_picture.size > 10.megabytes
   end
+
+  def lat_lng_unchanged
+    if self.persisted?
```

Comment:
@arunava59 This won't work, since there can be a state where user is saved but home location is yet to be updated. 

Instead you might wanna check if lat, lng already persisted. This can be a bit tricky since you can't just check for attributes , you need to check for persisted values. Please figure out how to do that

Reply:
Okay, I was under the assumption that we would be registering a new user in our database with his home location data.
I have updated the PR. Please check and tell if anything more is needed to be done.

Reply:
@arunava59 Looks good now. Thanks